src/com/palmergames/bukkit/towny/object/TownBlock.java

```diff
@@ -453,4 +462,34 @@ public long getClaimedAt() {
 	public void setClaimedAt(long claimedAt) {
 		this.claimedAt = claimedAt;
 	}
+
+	public Map<Resident, PermissionData> getPermissionOverrides() {
+		return permissionOverrides;
+	}
+
+	public List<Resident> getTrustedResidents() {
+		return trustedResidents;
+	}
+	
+	public boolean hasTrustedResident(Resident resident) {
+		return trustedResidents.contains(resident);
+	}
+	
+	public void addTrustedResident(Resident resident) throws AlreadyRegisteredException {
+		if (trustedResidents.contains(resident))
+			throw new AlreadyRegisteredException();
+		
+		trustedResidents.add(resident);
+	}
+	
+	public void removeTrustedResident(Resident resident) {
+		trustedResidents.remove(resident);
+	}
+	
+	public boolean hasResident(Resident resident) {
+		if (this.resident == null || resident == null)
+			return false;
+		
+		return resident.equals(this.resident);
+	}
```

Comment:
TODO: Figure out how we're going to handle permissionOverrides/Trusted residents on the PlotGroup level.